django/contrib/contenttypes/fields.py

```diff
@@ -31,6 +31,7 @@ class GenericForeignKey(object):
     one_to_many = False
     one_to_one = False
     related_model = None
+    originating_model = None
```

Comment:
Actually, the originating_model isn't needed any more, so I'm going to remove it.